client/packages/requisitions/src/ResponseRequisition/api/api.ts

```diff
@@ -209,6 +209,42 @@ export const getResponseQueries = (sdk: Sdk, storeId: string) => ({
 
     throw new Error('Unable to create requisition');
   },
+  insertRequestFromResponse: async ({
+    id,
+    responseRequisitionId,
+    otherPartyId,
+    comment,
+    maxMonthsOfStock,
+    minMonthsOfStock,
```

Comment:
Passing this in from frontend so we don't have to get requisition data again from backend =/ we already have this information anyways..